keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -253,14 +274,28 @@ nitty-gritty.
 -->
 
 ### Overview
-Each cluster in a ClusterSet will be assigned a unique identifier, that lives at least as long as that cluster is a member of the given ClusterSet, and is immutable for that same lifetime. This identifier will be stored in a new cluster-scoped `ClusterProperty` CR with the well known name `cluster.clusterset.k8s.io` that may be referenced by workloads within the cluster. The identifier must either:
+Each cluster in a ClusterSet will be assigned a unique identifier, that lives at
+least as long as that cluster is a member of the given ClusterSet, and is
+immutable for that same lifetime. This identifier will be stored in a new
+cluster-scoped `ClusterProperty` CR with the well known name
+`cluster.clusterset.k8s.io` that may be referenced by workloads within the
+cluster. The identifier must either:
 - be a valid [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS label,
-- or be composed of two valid [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS labels separated with a dot.
-The identifier may be created by an implementation dependent mechanism.
-
-While a member of a ClusterSet, a cluster will also have an additional `clusterset.k8s.io ClusterProperty` which describes its current membership. This property must be present exactly as long as the cluster's membership in a ClusterSet lasts, and removed when the cluster is no longer a member.
-
-More detail and examples of the uniqueness, lifespan, immutability, and content requirements for both the `cluster.clusterset.k8s.io ClusterProperty` and `clusterset.k8s.io ClusterProperty` are described further below. The goal of these requirements are to provide to the MCS API a cluster id of viable usefulness to address known user stories without being too restrictive or prescriptive.
+- or be composed of two valid [RFC-1123](https://tools.ietf.org/html/rfc1123)
+DNS labels separated with a dot. The identifier may be created by an
+implementation dependent mechanism.
+
+While a member of a ClusterSet, a cluster will also have an additional
+`clusterset.k8s.io ClusterProperty` which describes its current membership. This
+property must be present exactly as long as the cluster's membership in a
```

Comment:
s/exactly//  - it's impossible to do "exactly".  What about:

"""
This property must be present  while the cluster is a member of a ClusterSet, and must be removed when the cluster is no longer a member.
"""